app/javascript/layouts/app_router/types/AppRouter__Course.res

```diff
@@ -0,0 +1,58 @@
+type community = {
+  communityName: string,
+  communityId: string,
+}
+
+type t = {
+  id: string,
+  name: string,
+  review: bool,
```

Comment:
Maybe we should name it something similar to `isStudent` which has more clarity on purpose. `canReview` or `isCoach` likes.